docs/plotting.md

```diff
@@ -1,63 +1,79 @@
 # Plotting
+
 This page explains how to plot prices, indicator, profits.
 
 ## Installation
 
 Plotting scripts use Plotly library. Install/upgrade it with:
 
-```
-pip install --upgrade plotly
+``` bash
+pip install -U -r requirements-plot.txt
 ```
 
 At least version 2.3.0 is required.
 
 ## Plot price and indicators
+
 Usage for the price plotter:
```

Comment:
And how to plot indicators as the section says? Did not find this below...

Reply:
mmmh good point :+1: